_posts/2021-02-01-wszystko-co-chcielibyscie-wiedziec-o-fontach-ale-baliscie-sie-zapytac.md

```diff
@@ -0,0 +1,106 @@
+---
+layout:    post
+title:     Wszystko, co chcielibyście wiedzieć o fontach, ale baliście się zapytać
+published: false
+date:      2021-01-01 09:00:00 +0100
+author:    pgrobelny
+image:     
+tags:
+    - font
+---
+
+Większość osób, które pracowały na frontendzie na pewno miały do czynienia z fontami, czy to wrzucały je do projektu, 
+czy po prostu zmieniały w jaki sposób tekst wyświetla się na stronie. Na pierwszy rzut oka może się wydawać, że to bajecznie proste. 
+Jest jednak kilka ważnych rzeczy, na które warto zwracać uwagę i być ich świadomym.
+
+## Sposoby określania wielkości fontów (EM, REM, PX)
```

Comment:
Ja bym tą część trochę przeorganizował. Wydaje mi się, że fajnie by było zacząć opowieść od pixeli, dojść do problemów, które powodują i przedstawić EM/REM jako jedno z możliwych rozwiazań

Reply:
W sumie dobry pomysł, spróbuję to przerobić